CSS changes to decimalParts to allow copy paste.

buck54321 opened the pull request:
Resolves #885. 

Apparently, flexbox makes all children act as block elements in certain ways, one of which is that newlines are inserted on copy-paste between what would normally be inline elements like spans, regardless of the CSS `display` property and regardless of `!important` rules.

gozart1 commented on public/scss/mempool_vote_table.scss:
```diff
@@ -7,67 +7,3 @@
   background: #0078ff1c;
 }
-
-.mb-3 tbody tr td .fs14-decimal {
```
I'm glad you are deleting these horribly brittle selectors.
But it turns out that this one in particular keeps the 'DCR' aligned to the left in the home page:
<img width="1266" alt="screen shot 2018-12-17 at 11 57 05 pm" src="https://user-images.githubusercontent.com/25571523/50139827-a418a680-0257-11e9-8f26-f86a78e374e5.png">

But rather than author any styles to do this, you could just put d-flex on the wrapper div to align things

buck54321 commented:
Removed excess margins around DCR unit and corrected a couple of alignment issues in the last commit. I avoided flexbox for alignment in places where standard inline behavior fit the bill. 

Switched to template literals for `humanize.DecimalParts`.
